src/mc_control/fsm/states/Grippers.cpp

```diff
@@ -28,11 +29,28 @@ void Grippers::start(Controller & ctl)
         LOG_WARNING("[FSM::" << name() << "] " << g << " is not a known gripper")
         continue;
       }
-      auto gripper = ctl.grippers[g];
+      auto & gripper = ctl.grippers[g];
+      grippers(g)("percentVMAX", gripper->percentVMAX());
```

Comment:
This does nothing because `gripper->percentVMAX()` returns a value

You want:
```cpp
gripper->percentVMAX(grippers(g)("percentVMAX", gripper->percentVMAX()));
```